torch/onnx/symbolic_opset9.py

```diff
@@ -1800,33 +1812,51 @@ def le(g, input, other):
 
 
 def __and_(g, input, other):
-    if input.type().scalarType() == "Bool" and other.type().scalarType() == "Bool":
-        return g.op("And", input, other)
-    else:
-        raise NotImplementedError(
+    if input.type().scalarType() != "Bool":
+        raise errors.SymbolicValueError(
             "ONNX export does NOT support exporting bitwise AND "
-            + "for non-boolean input values"
+            "for non-boolean input values",
+            input,
         )
+    if other.type().scalarType() != "Bool":
+        raise errors.SymbolicValueError(
+            "ONNX export does NOT support exporting bitwise AND "
+            "for non-boolean input values",
+            other,
+        )
+    return g.op("And", input, other)
 
 
 def __or_(g, input, other):
-    if input.type().scalarType() == "Bool" and other.type().scalarType() == "Bool":
-        return g.op("Or", input, other)
-    else:
-        raise NotImplementedError(
+    if input.type().scalarType() != "Bool":
```

Comment:
Maybe we can pull out the if condition with a shared function? There are four of them.

Reply:
Done